docs/README.md

```diff
@@ -66,59 +66,61 @@ The following sections are Kubernetes specific. If you are a Kubernetes user, us
-| v0.2.0                 | amazon/aws-efs-csi-driver:v0.2.0 |
-| v0.1.0                 | amazon/aws-efs-csi-driver:v0.1.0 |
+|-------------------------------|----------------------------------|
+| master branch                 | amazon/aws-efs-csi-driver:master |
+| v1.6.0                        | amazon/aws-efs-csi-driver:v1.6.0 |
+| v1.5.9                        | amazon/aws-efs-csi-driver:v1.5.9 |
+| v1.5.8                        | amazon/aws-efs-csi-driver:v1.5.8 |
+| v1.5.7                        | amazon/aws-efs-csi-driver:v1.5.7 |                                  
+| v1.5.6                        | amazon/aws-efs-csi-driver:v1.5.6 |
+| v1.5.5                        | amazon/aws-efs-csi-driver:v1.5.5 |
+| v1.5.4                        | amazon/aws-efs-csi-driver:v1.5.4 |                                  
+| v1.5.3                        | amazon/aws-efs-csi-driver:v1.5.3 |
+| v1.5.2                        | amazon/aws-efs-csi-driver:v1.5.2 |
+| v1.5.1                        | amazon/aws-efs-csi-driver:v1.5.1 |
+| v1.5.0                        | amazon/aws-efs-csi-driver:v1.5.0 |
+| v1.4.9                        | amazon/aws-efs-csi-driver:v1.4.9 |
+| v1.4.8                        | amazon/aws-efs-csi-driver:v1.4.8 |
+| v1.4.7                        | amazon/aws-efs-csi-driver:v1.4.7 |
+| v1.4.6                        | amazon/aws-efs-csi-driver:v1.4.6 |
+| v1.4.5                        | amazon/aws-efs-csi-driver:v1.4.5 |
+| v1.4.4                        | amazon/aws-efs-csi-driver:v1.4.4 |
+| v1.4.3                        | amazon/aws-efs-csi-driver:v1.4.3 |
+| v1.4.2                        | amazon/aws-efs-csi-driver:v1.4.2 |
+| v1.4.1                        | amazon/aws-efs-csi-driver:v1.4.1 |
+| v1.4.0                        | amazon/aws-efs-csi-driver:v1.4.0 |
+| v1.3.8                        | amazon/aws-efs-csi-driver:v1.3.8 |
+| v1.3.7                        | amazon/aws-efs-csi-driver:v1.3.7 |
+| v1.3.6                        | amazon/aws-efs-csi-driver:v1.3.6 |
+| v1.3.5                        | amazon/aws-efs-csi-driver:v1.3.5 |
+| v1.3.4                        | amazon/aws-efs-csi-driver:v1.3.4 |
+| v1.3.3                        | amazon/aws-efs-csi-driver:v1.3.3 |
+| v1.3.2                        | amazon/aws-efs-csi-driver:v1.3.2 |
+| v1.3.1                        | amazon/aws-efs-csi-driver:v1.3.1 |
+| v1.3.0                        | amazon/aws-efs-csi-driver:v1.3.0 |
+| v1.2.1                        | amazon/aws-efs-csi-driver:v1.2.1 |
+| v1.2.0                        | amazon/aws-efs-csi-driver:v1.2.0 |
+| v1.1.1                        | amazon/aws-efs-csi-driver:v1.1.1 |
+| v1.1.0                        | amazon/aws-efs-csi-driver:v1.1.0 |
+| v1.0.0                        | amazon/aws-efs-csi-driver:v1.0.0 |
+| v0.3.0                        | amazon/aws-efs-csi-driver:v0.3.0 |
+| v0.2.0                        | amazon/aws-efs-csi-driver:v0.2.0 |
+| v0.1.0                        | amazon/aws-efs-csi-driver:v0.1.0 |
```

Comment:
Seems we have not been updating public ECR tags here, let' change it to v1.6.0?

Reply:
Sure, will do